certified-connectors/Scrive eSign/apiProperties.json

```diff
@@ -1,64 +1,143 @@
             "required": "false",
             "tabIndex": 4
-          },
-          "description": "Specify Scrive API host. Default is https://scrive.com",
-          "displayName": "API Host URL",
-          "tooltip": "Provide the host URL for the API."
+          }
         }
       }
     },
-    "policyTemplateInstances": [
-      {
-        "templateId": "setheader",
-        "title": "API host URL",
-        "parameters": {
-          "x-ms-apimTemplate-policySection": "Request",
-          "x-ms-apimTemplateParameter.existsAction": "override",
-          "x-ms-apimTemplateParameter.name": "customApiUrl",
-          "x-ms-apimTemplateParameter.value": "@connectionParameters('customApiUrl','https://scrive.com')"
+    "connectionParameterSets": {
+      "uiDefinition": {
+        "displayname": "Authentication Type",
+        "description": "Type of authentication to be used."
+      },
+      "values": [
+        {
+          "name": "Email/Password",
+          "uiDefinition": {
+            "displayname": "Email/password",
+            "description": "Basic authentication using a Scrive email and password."
+          },
+          "parameters": {
+            "username": {
+              "type": "securestring",
+              "description": "The Scrive account email.",
+              "displayName": "Email for Scrive account",
+              "tooltip": "Provide the email for the Scrive account",
+              "uiDefinition": {
+                "constraints": {
+                  "clearText": true,
+                  "required": "true",
+                  "tabIndex": 2
+                }
+              }
+            },
+            "password": {
+              "type": "securestring",
+              "description": "The Scrive account password.",
+              "displayName": "Password for Scrive account",
+              "tooltip": "Provide the password for the Scrive account",
+              "uiDefinition": {
+                "constraints": {
+                  "clearText": false,
+                  "required": "true",
+                  "tabIndex": 3
+                }
+              }
+            },
+            "customApiUrl": {
+              "type": "securestring",
+              "description": "Specify Scrive API host. Default is https://scrive.com.",
+              "displayName": "API Host URL",
+              "tooltip": "Provide the host URL for the API.",
+              "uiDefinition": {
+                "constraints": {
+                  "clearText": true,
+                  "required": "false",
+                  "tabIndex": 4
+                }
+              }
+            }
+          }
+        },
+        {
+          "name": "OAuth",
+          "uiDefinition": {
+            "displayname": "OAuth",
+            "description": "OAuth authentication."
+          },
+          "parameters": {
+            "token": {
+              "oAuthSettings": {
+                "clientId": "b656c82f946739f0_36551",
```

Comment:
@marcusdelang secret look alike values are not allowed. Please replace with dummy value.